packages/vendor-core/package.json

```diff
@@ -47,6 +48,8 @@
     "dsmorse-gridster": "^0.8.0",
     "file-saver": "^2.0.1",
     "formik": "^1.5.8",
+    "graphql": "^15.5.0",
+    "graphql-tag": "^2.11.0",
```

Comment:
Shouldn't it suppose to be supplied by `@apollo/client`?

Reply:
Correct, removed.